packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -429,15 +432,11 @@ end
       timeout: timeoutConfiguration.slowOperation,
     );
     try {
-      await _aotBuilder.build(
-        platform: TargetPlatform.ios,
-        outputPath: destinationDirectory.path,
-        buildInfo: buildInfo,
-        // Relative paths show noise in the compiler https://github.com/dart-lang/sdk/issues/37978.
-        mainDartFile: globals.fs.path.absolute(targetFile),
-        quiet: true,
-        bitcode: true,
-        iosBuildArchs: <DarwinArch>[DarwinArch.armv7, DarwinArch.arm64],
+      await _buildAotFrameworkWithAssemble(
+        buildInfo,
+        destinationDirectory,
+        targetFile,
+        true,
```

Comment:
Don't need bitcode as a param to `_buildAotFrameworkWithAssemble`, set true directory in Environment 
 `kBitcodeFlag`